guard/src/rules/eval/operators.rs

```diff
@@ -115,18 +106,18 @@ pub(crate) struct NotComparable<'r> {
     pub(crate) pair: LhsRhsPair<'r>,
 }
 
-pub(super) fn resolved<'value, E, R>(
-    qr: &QueryResult<'value>,
-    err: E,
-) -> std::result::Result<&'value PathAwareValue, R>
-where
-    E: Fn(UnResolved<'value>) -> R,
-{
-    match qr {
-        QueryResult::Resolved(r) | QueryResult::Literal(r) => Ok(*r),
-        QueryResult::UnResolved(ur) => Err(err(ur.clone())),
-    }
-}
+// pub(super) fn resolved<'value, E, R>(
```

Comment:
Why is this chunk commented out?

Reply:
same as the previous commented out chunk